packages/guardian/guardian.0.0.4/opam

```diff
@@ -0,0 +1,60 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Role-based access control for OCaml"
+description:
+  "Defined actors can access targets according to specified rules (allowed actions: CRUD)."
+maintainer: ["engineering@econ.uzh.ch"]
+authors: [
+  "Department of Economics, University of Zurich" "John Christopher McAlpine"
+]
+license: "GPL-2.0-or-later"
+homepage: "https://github.com/uzh/guardian"
+doc: "https://github.com/uzh/guardian"
+bug-reports: "https://github.com/uzh/guardian/issues"
+depends: [
+  "base64" {>= "3.5.0"}
+  "caqti-driver-mariadb" {>= "1.8.0"}
+  "caqti-lwt" {>= "1.8.0"}
+  "conformist" {>= "0.8.1"}
+  "containers" {>= "3.6"}
+  "dune" {>= "2.9"}
+  "logs" {>= "0.7.0"}
+  "lwt" {>= "5.6.1"}
+  "lwt_ppx" {>= "2.1.0"}
+  "mariadb" {= "1.1.4"}
```

Comment:
oh i missed that constraint. Could you open a PR to change the constraint to something less strict like `>= 1.1.4` ?

Reply:
updated here: https://github.com/ocaml/opam-repository/pull/22567